<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>437</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0511364 rgba(20, 59, 79, 255), stop:1 rgba(96, 104, 146, 246));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="decode">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 88, 145);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Dekoduoti</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Dekoduotas tekstas atsiras čia&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="about">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>96</green>
          <blue>166</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>96</green>
          <blue>166</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>34</red>
          <green>88</green>
          <blue>145</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>96</green>
          <blue>166</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 88, 145);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Apie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encodedMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 88, 145);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Užkoduoti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decodedMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 88, 145);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Dekoduoti</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="textLength">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(17, 60, 83);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-8</y>
      <width>141</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(17, 60, 83);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(17, 60, 83);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Erdvę užpildančios kreivės</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 88, 145);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Ieškoti</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imagename">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,255)
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imagepath">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="compare">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 88, 145);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Palyginti paveikslėlius</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>189</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Įveskite paveiksliuko, kurį &lt;br/&gt;norite atkoduoti, pavadinimą&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Įveskite užkoduoto teksto ilgį&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="aboutBrowser">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>581</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0,0,0,0%);
border-top-color: rgb(0,0,0,0%);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; color:#ffffff;&quot;&gt;Tai bakalauro darbas, kurio pagrindinis tikslas yra panaudoti erdvę užpildančias tieses duomenų slėpimui paveikslėliuose.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:11pt; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;Kaip ši programa veikia?&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; color:#ffffff;&quot;&gt;&lt;br /&gt;Užkoduoti skiltyje mes galime pasirinktame paveikslėlyje užkoduoti norimą tekstą. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; color:#ffffff;&quot;&gt;Pasirinkimus dekodavimo skiltį, reikia jau turėti užkoduotą paveiksliuką. Norint dekoduoti, taip pat reikia žinoti teksto ilgį, kitaip dekoduoti nebus galima. &lt;br /&gt;Taip pat sukurta yra ir paveikslėlių palyginimo skiltis, kurioje galime matyti, kaip užkoduotas paveikslėlis skirsis nuo pradinio mūsų paveikslėlio.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Įveskite norimą užkoduoti tekstą&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encode">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 88, 145);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Užkoduoti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compareImage">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 88, 145);
gridline-color: rgb(202, 202, 202);
selection-color: rgb(85, 255, 255);
selection-background-color: rgb(39, 96, 166);</string>
    </property>
    <property name="text">
     <string>Palyginti paveikslėlius</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0%)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Įveskite užkoduoto paveiksliuko&lt;br/&gt;pavadinimą&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="decodedname">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,255)
</string>
    </property>
   </widget>
   <zorder>textEdit</zorder>
   <zorder>encode</zorder>
   <zorder>label1</zorder>
   <zorder>imagepath</zorder>
   <zorder>label_3</zorder>
   <zorder>decode</zorder>
   <zorder>result</zorder>
   <zorder>about</zorder>
   <zorder>encodedMenu</zorder>
   <zorder>decodedMenu</zorder>
   <zorder>textLength</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>browse</zorder>
   <zorder>imagename</zorder>
   <zorder>compare</zorder>
   <zorder>labelImage</zorder>
   <zorder>label</zorder>
   <zorder>compareImage</zorder>
   <zorder>labelName</zorder>
   <zorder>decodedname</zorder>
   <zorder>aboutBrowser</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
